app/team.go

```diff
@@ -701,7 +701,7 @@ func (a *App) LeaveTeam(team *model.Team, user *model.User, requestorId string)
 
 	var channelList *model.ChannelList
 
-	if result := <-a.Srv.Store.Channel().GetChannels(team.Id, user.Id); result.Err != nil {
+	if result := <-a.Srv.Store.Channel().GetChannels(team.Id, user.Id, false); result.Err != nil {
```

Comment:
When you leave a team, you must leave the Archive channels too?

Reply:
Yeah I think so. We don't have a `DeleteAt` concept for `ChannelMembers`, so on leaving a team we just delete them all. I think it makes sense to include archived channels in that too (within the constraints of our existing data model.

Reply:
Agreed unless something comes up that is a good argument for breaking the model, best to keep it the same.

Reply:
Done! :)